content/en/docs/Examples/services.md

```diff
@@ -913,6 +1095,34 @@ service:
       icon: https://raw.githubusercontent.com/rancher/docs/master/static/imgs/rancher-logo-cow-blue.svg
 ```
 
+## Readarr/Readarr
+Source: https://github.com/Readarr/Readarr
+
+- deployed_version - Requires an `API_KEY` which can be retrieved at `Settings/General/Security/API Key`
+```yaml
+service:
+  Readarr:
+    options:
+      semantic_versioning: false
+    latest_version:
+      type: github
+      url: Readarr/Readarr
+      use_prerelease: true
+      url_commands:
+        - type: regex
+          regex: v([0-9.]+)$
+    deployed_version:
+      method: GET
+      url: https://readarr.example.io/api/v1/system/status
+      headers:
+        - key: X-Api-Key
+          value: <API_KEY>
+      json: version
+    dashboard:
+      icon: https://raw.githubusercontent.com/walkxcode/dashboard-icons/main/png/readarr.png
```

Comment:
```suggestion
      icon: https://raw.githubusercontent.com/Readarr/Readarr/develop/Logo/1024.png
```
Would prefer to keep it all in the same repo. I don't mind if you'd prefer the Readarr.svg in that dir (some things have trouble with SVG, so I kept it as a PNG like you had)